### DISCUSSION TITLE
Nit: custom height for tabs

### DISCUSSION THREAD

**Discussion by ztomer** [ghostty-org/ghostty#3786] [Feature Requests, Ideas] [answered]
A minor nit: is it possible to set a custom height for the tabs? in ITerm2 there's a whole additional line, so it's like, super critical to fix this (*) 

Left is iTerm2, right is Ghostty

<img width="963" alt="image" src="https://github.com/user-attachments/assets/0a882cea-868b-45be-afa6-b55c616f286b" />

(*) For at least two, maybe even three people.

**Comment by JakeWharton** [comment, marked as answer]
Duplicate of #3297
